back-end/routers/productsRoute.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const router = express.Router();
+
+const { authenticate } = require("../services/productsService");
+const { generalResponse } = require("../helpers/responses");
+const { mockData, mockError } = require("../helpers/mock-data");
```

Comment:
@Jeffzholy make your life easier with `const mock = require(...);` so you can refer to it with `mock.data` and `mock.error`